updater/lib/dependabot/api_client.rb

```diff
@@ -290,8 +290,6 @@ def create_pull_request_data(dependency_change, base_commit_sha)
         "base-commit-sha": base_commit_sha
       }.merge(dependency_group_hash(dependency_change))
 
-      return data unless dependency_change.pr_message
```

Comment:
One of the more interesting things Sorbet caught was that pr_message is never nil because it's actually a class instance that always can be created.